src/core/Navigation/SideNavigation/SideNavigationItem/SideNavigationItem.baseStyles.tsx

```diff
@@ -0,0 +1,206 @@
+import { font } from '../../../theme/reset';
+import { css } from 'styled-components';
+import { SuomifiTheme } from '../../../theme';
+
+export const baseStyles = (theme: SuomifiTheme) => css`
+  ${font(theme)('bodySemiBold')};
+
+  /* stylelint-disable no-descending-specificity */
+  /* Nested :hover etc selectors do not work well with this rule. */
+  &.fi-side-navigation-item {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    position: relative;
+
+    &:after {
+      content: '';
+      position: absolute;
+      bottom: -1px;
+      left: 2.5%;
+      width: 95%;
+      height: 1px;
+      background: ${theme.colors.depthSecondary};
+    }
```

Comment:
What is the purpose of this exactly? When colored red, it looks like this: 
![image](https://user-images.githubusercontent.com/54316341/192996151-54f6bc72-f9d5-4465-8d0e-363d661a411f.png)

Reply:
Seems to have been a display issue for me. Couldn't distinguish the lines with laptop screen at first. However, this colorization seems to have highlighted a small issue with the separator lines.